docs/date.md

```diff
@@ -1,96 +1,398 @@
+- any object with properties denoting a duration, such as `{ days: 5 }`;
+- a string in ISO 8601 duration format, such as `P5D`.
+
+However, the `duration` argument must be in whole days.
+For example, `{ days: 5, hours: 0 }` is allowed, but `{ days: 5, hours: 12 }` will throw a `RangeError`.
+
+Some subtractions may be ambiguous, because months have different lengths.
+For example, subtracting one month from July 31 would result in June 31, which doesn't exist.
+For these cases, the `disambiguation` argument tells what to do:
+- In `constrain` mode (the default), out-of-range values are clamped to the nearest in-range value.
+- In `balance` mode, out-of-range values are resolved by balancing them with the next highest unit.
+- In `reject` mode, an addition that would result in an out-of-range value fails, and a `RangeError` is thrown.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.minus({years: 20, months: 4})  // => 1986-04-24
+date.minus('P14Y8D')  // => 1992-08-16
+
+date = Temporal.Date.from('2019-03-31')
+date.minus({months: 1}, 'constrain')  // => 2019-02-28
+date.minus({months: 1}, 'balance')  // => 2019-03-03
+date.minus({months: 1}, 'reject')  // => throws
+```
 
 ### date.**difference**(_other_: Temporal.Date) : Temporal.Duration
 
-Returns a new `Duration` object which is the difference between the current `Date` object and the argument `Date` value.
+**Parameters:**
+- `other` (`Temporal.Date`): Another date with which to compute the difference.
+
+**Returns:** a `Temporal.Duration` representing the difference between `date` and `other`.
+
+This method computes the difference between the two dates represented by `date` and `other`, and returns it as a `Temporal.Duration` object.
+The difference is always positive, no matter the order of `date` and `other`, because `Temporal.Duration` objects cannot represent negative durations.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.difference(Temporal.Date.from('2019-01-31'))  // => P12Y5M7D
+```
 
 ### date.**toString**() : string
 
-Returns an ISO 8601 string representing the current `Date` object
+**Returns:** a string in the ISO 8601 date format representing `date`.
+
+This method overrides the `Object.prototype.toString()` method and provides a convenient, unambiguous string representation of `date`.
+The string can be passed to `Temporal.Date.from()` to create a new `Temporal.Date` object.
+
+Example usage:
+```js
+date = Temporal.Date.from('2006-08-24');
+date.toString();  // => 2006-08-24
+```
 
 ### date.**toLocaleString**(_locale_?: string, _options_?: object) : string
 
-Returns a string with a locally sensitive representation of the specified `Date` object. Overrides the `Object.prototype.toLocaleString()` method.
+**Parameters:**
+- `locales` (optional string or array of strings): A string with a BCP 47 language tag with an optional Unicode extension key, or an array of such strings.
```

Comment:
If you haven't already done so, you can copy the boilerplate that we have on, e.g., `Date.prototype.toLocaleString`.

Reply:
For now I've included an abridged version of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat with a link to the whole thing. I do think we should include more complete information on each doc page eventually, but I'd like to see how the implementation of calendars ends up affecting this method before going into more detail.

Reply:
Ok, thanks